tests/test_pdb_tidy.py

```diff
@@ -64,13 +64,18 @@ def test_default(self):
 
         # Validate results
         self.assertEqual(self.retcode, 0)  # ensure the program exited OK.
-        self.assertEqual(len(self.stdout), 207)
+        # CONECTs are ignored by issue #72, expected only 205 lines
+        self.assertEqual(len(self.stdout), 205)
```

Comment:
This check is unnecessary if you check explicitly for the number of CONECT statements someplace else no?

Reply:
I disagree,

imagine the changes I just made in this PR mess up with the number of lines for any other reason? Also, this test was not written now, I just updated the numbers.

I believe both tests are valid. There are never too many tests ;-)

Reply:
There _is_ such a thing as too many tests, but OK, add it as a good sanity check :)

Reply:
Yet to be found :hand_over_mouth: